Spring clean on Getting started docs section (#5999)
PostHog/posthog.com: contents/docs/getting-started/group-analytics.mdx

import ChatGroupType from "./\_snippets/group-type-create-chat.mdx"
import SetGroupProperties from "./\_snippets/set-group-properties.mdx"

> **Note: ** This is a paid feature and is not available on the open source or free cloud plan. See our [pricing page](/pricing) for more.

Group analytics is a powerful feature in PostHog that allows you to perform analytics on entities (e.g. companies, teams etc.) other than single users.
This feature is slightly more advanced, but opens up entirely new avenues for analytics that are essential for most B2B use cases.

Group analytics can sometimes feel confusing to new users, and this often comes from skipping over the basic concepts too quickly.

> We'll start with an overview of how Group analytics works, and then jump into creating new groups and viewing them within PostHog.

Reviewer: Groups are often confused with Cohorts. There needs to be a callout here linking to the cohorts docs, with a brief sentence on how they're different.  I know theres the section below, but I think it's good to also call it out here at the intro

## 1. What is a group?

> In PostHog, a 'Group' is an entity that we can send events and hold properties that is beyond a single user.

Author: ```suggestion
In PostHog, a 'Group' is an entity that we can send events and hold properties for that is beyond a single user.
```

Reviewer: The phrase "that is beyond a single user". What does this mean? We should clarify

> Here are two examples of common entities we can model using groups:

Reviewer: I dont like the use of the term `model` here. It's a new term that we dont use anywhere else in docs, and it's not clear what we mean.

I think better would be `Here are two examples of common entities we can organize events by using groups:`
(there may be a better word for "organize". I was trying to avoid saying `we can group events by using groups`

> 1. **Organizations/Teams**: It's very common in B2B apps to have multiple users under the same 'Team' account, in which case groups allow us to perform analytics on individual teams as if they were users.

Reviewer: Im not clear what we mean by " as if they were users.". Also, is "perform" the correct verb for analytics? Maybe Investigate?

```suggestion
1. **Organizations/Teams**: It's very common in B2B apps to have multiple users under the same 'Team' account, in which case groups allow us to investigate analytics on individual teams.
```

2. **Threads/Chats/Posts**: If you're creating a social app, you might want to track events that are connected with specific conversations, such as replies.

Looking at these examples, it might be tempting to think that a group is just a group of users; however, this is not exactly the case.

> We'll see this later, but the basic idea is that **only events are connected to groups directly, and any user who sent one of those events will also be linked indirectly**.

Author: Yeah, I have no idea!

> In this way, there isn't a way to directly 'assign' a user to a group - instead, this user would send an event with a special property linking the event to a group.

Author: Interested in your input here @Lior539 / @ivanagas.

Perhaps it's my lack of technical background, but this explanation of groups is quite confusing to me. Is there a better way to explain this?

Reviewer: I dont know what this sentence means tbh.

It could be worth calling out cohorts here again e.g.

`there isn't a way to directly 'assign' a user to a group – but you should use cohorts for that` (but phrase it better)
